### ISSUE TITLE
logger-typescript dereferences undefined

### ISSUE THREAD

**Issue by ataraxus** [driftyco/ionic-app-scripts#501]
#### Short description of the problem:
Transpiling breaks with 
```
> ionic-app-scripts serve

[10:16:43]  ionic-app-scripts 0.0.46
[10:16:43]  watch started ...
[10:16:43]  build dev started ...
[10:16:43]  clean started ...
[10:16:43]  clean finished in 8 ms
[10:16:43]  copy started ...
[10:16:43]  transpile started ...
[10:16:50]  build dev failed: Cannot read property 'indexOf' of undefined
```

#### What behavior are you expecting?
transpiling to give a hint whats broken

**Steps to reproduce:**
I can't really reproduce it... but I found the faulting line: 
```
       // my debug stuff
        if(!errorLine.html)
            console.log("Gotcha!3", errorLine);
        if (errorLine.html.indexOf('class="hljs') === -1) {
            try {
                errorLine.html = highlight_1.highlight(d.language, errorLine.text, true).value;
            }
            catch (e) { }
        }
```
which than results in something like this: 

```
Gotcha!3 { lineIndex: 905,
  lineNumber: 906,
  text: undefined,
  html: undefined,
  errorCharStart: 54,
  errorLength: 1 }
```

When i move around the faulting line transpiling shows me the error in HTML but I see it points to an old HTML reprentation of my code. hence i suspect that the process of generating the html view of my code is somehow broken, _if_ the offending line is than at the end of the document which got bigger, it will point to a line not existing in the html. 

**Which @ionic/app-scripts version are you using?**
```
Your system information:

 ordova CLI: 6.3.1
Ionic Framework Version: 2.0.0-rc.3
Ionic CLI Version: 2.1.13
Ionic App Lib Version: 2.1.7
Ionic App Scripts Version: 0.0.46
ios-deploy version: Not installed
ios-sim version: Not installed
OS: Windows 7
Node Version: v6.9.1
Xcode version: Not installed

```

**Comment by danbucholtz** [comment]
@ataraxus,

This is very helpful. Thanks! What are you doing to cause the issue?

Thanks,
Dan

**Comment by ataraxus** [comment]
Hey Dan, sorry that I can't give a guide for reproducing this failure. I think it would be still unnoticed, if my syntax error didn't happen on the last line....

I'll try later to reproduce it, but since I tried hard the first time, so I don't think I'll manage it this time. Feel free to close this issue. :(

**Comment by danbucholtz** [comment]
Hi all,

This is resolved in `0.0.48` which was just published.

Please let me know if you have any issues.

Thanks,
Dan

**Comment by ataraxus** [comment]
Great! So you could actually reproduce and fix the issue? Kudos
